Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/openjdk:8u91-b14-1-22
+FROM registry.opensource.zalan.do/stups/openjdk:8-24
```

Comment:
@sarnowski did this (without notifying), but he is right: we update the JDK version by hand in the pipeline, so the JDK build number might be wrong (tag vs. actual image contents) --- better to have only the major version which cannot have a "mismatch"